conf/messages

```diff
@@ -179,7 +180,8 @@ estimatedLPP1.amountReceived = Amount received
 estimatedLPP1.leftToPay = Left to pay
 estimatedLPP1.warning = The penalty will increase by a further {0}% of the unpaid VAT, if VAT remains unpaid {1} days after the due date.
 estimatedLPP1.estimatesHeading = Estimates
-estimatedLPP1.estimatesSubHeading = Penalties will show as estimates until:
+estimatedLPP1.estimatesSubHeading = Penalties will show as estimates until you pay the charge they relate to.
```

Comment:
You shouldn't be changing anything to do with LPP1 or LPP2. This task is for estimated interest only.

Reply:
Reverted back to original code